google-cloud-storage/lib/google/cloud/storage/file.rb

```diff
@@ -2081,6 +2081,50 @@ def self.gapi_from_attrs gapi, attributes
           Google::Apis::StorageV1::Object.new(**attr_params)
         end
 
+        ##
+        # from_gs_url is a method to fetch bucket details and file details from a gs url
+        #
+        # @return [Hash(String => String)]
+        #
+        # @example Fetch bucket_name and file_name from gs url:
+        #   require "google/cloud/storage"
+        #   gs_url= "gs://my-todo-app/avatars/heidi.jpeg"
+        #   file=Google::Cloud::Storage::File
+        #   file.from_gs_url(gs_url)
+        #   =>
+        #   {"bucket_name"=>"my-todo-app", "file_name"=>"avatars/heidi.jpeg"}
```

Comment:
`g/file_name/file_path`

Reply:
sorry I did not understand this comment can you explain a bit more

Reply:
Since we're not returning `file_name` anymore. So, we should rename `file_name` with `file_path` everywhere in this code.